content/departments/ce-support/support/process/how-to-tickets.md

```diff
@@ -0,0 +1,26 @@
+# Engaging Engineering
+
+As an Application Engineer, you will be confronted with tickets that require you to seek help from engineering. You are encouraged to do so! There are a two ways to enlist the help of engineering, largely depending on the nature of the ticket.
+
+### "How To" Tickets
+
+For how-to tickets, you are not required to create a Request for help (RFH) in GitHub. While RFH's are a valuable tool when dealing with complex issues, they introduce overhead and complexity that isn't necessary for "How To" questions. Furthermore, they limit visibility, and collaboration between teams that happens in public, shared channels.
+
+For "How To" tickets, follow the procedure below:
+
+1. As you normally would, check the [docs](https://docs.sourcegraph.com), and [ticket database](https://github.com/sourcegraph/support-tools-internal/tree/main/resolved-tickets) (searchable on `cse-k8s`) for possible solutions.
+2. If no solution is found, create a new post in the #customer-support-internal channel in Slack. Start the post with "HTQ:" so that it is searchable in the future.
+3. If you still don't have a solution after doing so, create a new post in #ask-engineering in slack. Again, make sure you start the post with "HTQ:" to make it more searchable. Post expiration will be turned off in this channel to accomodate searching for these posts and adding them to "How To" summaries.
```

Comment:
Post expiration (message retention) is set to infinite for all channels, actually. I know it used to be different, but it was extended for all channels a few months ago I think. I updated the handbook a couple weeks ago (https://sourcegraph.com/search?q=context:global+repo:sourcegraph/handbook%24+type:diff+slack+retention&patternType=literal).

```suggestion
3. If you still don't have a solution after doing so, create a new post in #ask-engineering in slack. Again, make sure you start the post with "HTQ:" to make it more searchable.
```